tests/testthat/setup.R

```diff
@@ -0,0 +1 @@
+Sys.setlocale("LC_MESSAGES", "C")
```

Comment:
Why did you add this setup file?

Reply:
Because otherwise the tests don’t run on my localized system (the german error messages don’t match the expected english ones)

Technically that change should go into another PR, but that’s a formality, so I thought I’ll only bother if you insist.

Reply:
Interesting, I wonder if this should be set by testthat